packages/core/src/external/aws/document-signing/bulk-sign.ts

```diff
@@ -64,60 +76,65 @@ export async function getSignedUrls(
 
     const response = urls.filter(url => url !== undefined) as DocumentBulkSignerLambdaResponse[];
 
-    await ossApiClient.callInternalEndpoint({
-      cxId: cxId,
-      patientId: patientId,
-      requestId: requestId,
-      dtos: response,
-      status: "completed",
-    });
+    await executeWithNetworkRetries(() =>
+      ossApiClient.callInternalEndpoint({
```

Comment:
this is double wrapping executeWithNetworkRetries
the callInternalEndpoint already uses executeWithNetworkRetries

Reply:
Good catch! I was concerned that might have left some like this, as I found the callInternalEndpoint fn after I did a bunch of changes. Thanks!